apps/api/src/app/blueprint/e2e/consts.ts

```diff
@@ -0,0 +1,89 @@
+import { INotificationTemplate, StepTypeEnum } from '@novu/shared';
+
+const STEPS = [
```

Comment:
I thought we could reuse the content of `apps/api/src/app/blueprint/usecases/get-grouped-blueprints/consts.ts` instead of duplicating hardcoded data. What do you think?

Reply:
Actually, I think we can, I will check it up after I will updat e the usecase costs.